pkg/backup/backup/backup_tracker.go

```diff
@@ -53,11 +55,31 @@ type trackDepends struct {
 
 // NewBackupTracker returns a BackupTracker
 func NewBackupTracker(deps *controller.Dependencies, statusUpdater controller.BackupConditionUpdaterInterface) BackupTracker {
-	return &backupTracker{
+	tracker := &backupTracker{
 		deps:          deps,
 		statusUpdater: statusUpdater,
 		logBackups:    make(map[string]*trackDepends),
 	}
+	go tracker.initTrackLogBackupsProgress()
+	return tracker
+}
+
+// initTrackLogBackupsProgress lists all log backups and track their progress.
+func (bt *backupTracker) initTrackLogBackupsProgress() {
+	backups, err := bt.deps.Clientset.PingcapV1alpha1().Backups("").List(context.TODO(), metav1.ListOptions{})
+	if err != nil {
+		klog.Warningf("list backups error %v, skip track all log backups progress when init, will track when log backup start", err)
+		return
+	}
+	klog.Infof("list backups size %d", len(backups.Items))
+	for _, backup := range backups.Items {
+		if backup.Spec.Mode == v1alpha1.BackupModeLog {
+			err = bt.StartTrackLogBackupProgress(&backup)
+			if err != nil {
```

Comment:
It seems it can not be restarted if StartTrackLogBackupProgress failed?

Reply:
yes, maybe can add retry, but if it is failed here, it will not affect normal started log backup.